libs/kotaemon/kotaemon/loaders/html_loader.py

```diff
@@ -78,6 +80,7 @@ class MhtmlReader(BaseReader):
 
     def __init__(
         self,
+        cache_dir: Optional[str] = flowsettings.KH_MARKDOWN_OUTPUT_DIR,
```

Comment:
Should be `getattr(flowsettings, "KH_MARKDOWN_OUTPUT_DIR", None)` to avoid error.